target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -14,6 +14,7 @@ ath79_setup_interfaces()
 	alfa-network,ap121f|\
 	aruba,ap-105|\
 	avm,fritz1750e|\
+	avm,fritzdvbc|\
```

Comment:
Please apply alphabetic sorting correctly (might change if you adjust the compatible)

Reply:
AFAIK numbers come before letters in most sorting methods.

Reply:
Yes, so avm,fritzdvbc should be after avm,fritz300e ...

Reply:
Oh, my comment was based on the snippet shown on GitHub where this wasn't visible. Sorry!